src/index.js

```diff
@@ -151,19 +153,21 @@ export default class ImageTool {
 
   /**
    * Renders Block content
+   *
    * @public
```

Comment:
such annotations do not make a sense because all class functions are public whatever we write, even if write `private` it won't make them private

Reply:
it is an example plugin, so these annotations intended for developers for better understanding what is a Block API and what is tool stuff that can be implemented by their own way

Reply:
But it is not about OOP principles and the Block's API is written in EditorJS documentation. These annotations are more abstract stuff.